web/app/js/components/ErrorModal.jsx

```diff
@@ -145,15 +145,17 @@ class ErrorModal extends React.Component {
         <Tooltip title="Pods are initializing"><CircularProgress size={20} thickness={4} /></Tooltip>
       );
     } else {
-      return <ErrorIcon onClick={this.handleClickOpen} />;
+      return (
+        <ErrorIcon color="error" fontSize="small" onClick={this.handleClickOpen} />
```

Comment:
Similar to my other comment -- would it be possible to make the height of this icon match the height of the text here, and remove the padding on the bottom?

![image](https://user-images.githubusercontent.com/9226/47394699-abe52000-d6d8-11e8-8568-3afdfe598ca2.png)

Reply:
That's so strange... it's aligned for me:
![screen shot 2018-10-24 at 11 00 50 am](https://user-images.githubusercontent.com/549258/47451244-20bd6600-d77c-11e8-87b0-b66136116daa.png)

Reply:
I figured it out, I was looking at the icons in the StatusTable, but the ones on the MetricsTable aren't aligned!

Reply:
Ahhh, that makes sense.